openpype/pype_commands.py

```diff
@@ -40,13 +40,20 @@ def launch_eventservercli(*args):
         )
         return run_event_server(*args)
 
+    @staticmethod
+    def launch_webpublisher_webservercli(*args, **kwargs):
+        from openpype.modules.webserver.webserver_cli import (
+            run_webserver
+        )
+        return run_webserver(*args, **kwargs)
+
     @staticmethod
     def launch_standalone_publisher():
         from openpype.tools import standalonepublish
         standalonepublish.main()
 
     @staticmethod
-    def publish(paths, targets=None):
+    def publish(paths, targets=None, host=None):
```

Comment:
`host` not used?

Reply:
I think this got mixed up from different PR.